projects/shared/src/lib/view/thumbnail/thumbnail.component.html

```diff
@@ -0,0 +1,28 @@
+<!--
+ RERO ILS UI
+ Copyright (C) 2019 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<div class="align-self-start d-flex justify-content-center">
+  <figure class="mb-0" [ngClass]="figureStyle">
+    <ng-container *ngIf="record.metadata.cover_art; else noLink">
+      <a href="coverUrl"><img [ngClass]="imageCssClass" [src]="coverUrl"></a>
+    </ng-container>
+    <ng-template #noLink>
+      <img [ngClass]="imageCssClass" [src]="coverUrl">
+    </ng-template>
+    <figcaption class="text-center">{{ record.metadata.type | translate }}</figcaption>
```

Comment:
`<figcaption class="text-center" transate>{{ record.metadata.type }}</figcaption>`

Reply:
`transate`is for holidays, no? Ok I leave ->[]